recipes/rsat-core/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+RSAT_DEST="$PREFIX/opt/rsat/"
+mkdir -p "$RSAT_DEST"
+mkdir -p "$PREFIX/bin"
+mkdir -p "$PREFIX/share/rsat"
+
+cp -a LICENSE.txt perl-scripts python-scripts makefiles R-scripts "$RSAT_DEST"
+cp bin/rsat $PREFIX/bin/rsat
+cp share/rsat/rsat.yaml $PREFIX/share/rsat/rsat.yaml
+
+# Build and dispatch compiled binaries
+# cd contrib
+for dbin in info-gibbs 
```

Comment:
LGTM, but I guess @jvanheld want to include the other targets as well?

Reply:
Indeed, so far I commented almost everything i order to have something that would compile. I will now 

- compile the other C tools
- include them in the test
- add test for the other tools (Perl, python)
- compile the internal R package
- run a test for a workflow (peak-motifs) and a tool (matrix-clustering) that combine more or less everything 

If these tests pass, I guess I would consider that the suite is running fine :-)